<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Далее</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="testProgressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>591</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v/%m</string>
    </property>
   </widget>
   <widget class="QLabel" name="pictureLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ярлык для битмап картинки</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="testNameLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ярлык для названия теста</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>671</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="answersLayout"/>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionTextLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>85</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ярлык для содержания вопроса</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="testsListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="backPanelWidget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>691</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="studNameField">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="resultsLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Результаты</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="studNameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО Студента:</string>
    </property>
   </widget>
   <zorder>backPanelWidget</zorder>
   <zorder>nextButton</zorder>
   <zorder>testProgressBar</zorder>
   <zorder>pictureLabel</zorder>
   <zorder>testNameLabel</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>startButton</zorder>
   <zorder>questionTextLabel</zorder>
   <zorder>testsListWidget</zorder>
   <zorder>resultsLabel</zorder>
   <zorder>studNameLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
